src/code/autolaunch/autolaunch.ts

```diff
@@ -24,14 +24,14 @@ function getURLParam (name) {
 // Adds query params to CODAP urls
 // this was taken from the document server `documents_v2_helper.rb`
 // We could also add documentServer parameter, or check query params
-function codap_v2_link(codapServer) {
+function codap_v2_link(codapServer: any) {
   const defaultCodapUrl = "https://codap.concord.org/releases/latest/"
   const documentServer = "https://document-store.concord.org/"
   const extraData = {
     documentServer
   }
   const baseUrl = codapServer || defaultCodapUrl
-  const {url, query, fragmentIdentifier} = queryString.parseUrl(baseUrl, {fragmentIdentifier: true})
+  const {url, query, fragmentIdentifier} = queryString.parseUrl(baseUrl, {parseFragmentIdentifier: true})
```

Comment:
This was a bug identified by the TypeScript conversion: `fragmentIdentifier` is not a valid option to `parseUrl()`.